src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,45 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
-);
+export const MovieCard = ({ movie }) => {
+  const { imgUrl, title, description, imdbUrl } = movie;
```

Comment:
if you have more than 3 props, more correctly place it in column, it will be more cleaner